src/store/reducers/tablets.ts

```diff
@@ -43,3 +45,31 @@ export const tabletsApi = api.injectEndpoints({
     }),
     overrideExisting: 'throw',
 });
+
+const getTabletsInfoSelector = createSelector(
+    (path: string) => path,
+    (path) => tabletsApi.endpoints.getTabletsInfo.select({path}),
+);
+
+const selectGetTabletsInfo = createSelector(
+    (state: RootState) => state,
+    (_state: RootState, path: string) => getTabletsInfoSelector(path),
+    (state, selectGetNodeStructure) => selectGetNodeStructure(state).data,
+);
+
+export const selectTabletsWithFqdn = createSelector(
+    (state: RootState, path: string) => selectGetTabletsInfo(state, path),
+    (state: RootState) => selectNodesMap(state),
+    (data, nodesMap): (TTabletStateInfo & {fqdn?: string})[] => {
+        if (!data?.TabletStateInfo) {
+            return [];
+        }
+        if (!nodesMap) {
+            return data as any;
```

Comment:
Why you return `any` here? It may be just `data.TabletStateInfo`, in case `viewer/json/nodelist` (`nodesMap`) will be responding too long, it will be just a table without fqdns. But in your case some problems with `nodelist` could break table